.gitignore

```diff
@@ -27,6 +27,8 @@ yarn-error.log*
 
 # local env files
 .env.local
+.env.test
+.env.development
```

Comment:
`.env.development` and `.env.test` don't need to be in gitignore

Reply:
I thought that it will contain, secret keys which are not to be exposed directly, that is why I had added it in git.ignore file

Reply:
No, those are not secret keys
Nothing is secret on frontend 😂